tests/UserActionTest.php

```diff
@@ -154,7 +153,7 @@ public function testScope2()
     public function testScope3()
     {
         $client = new ACClient($this->pers);
-        $client->addAction('new_client', ['scope' => UserAction\Generic::NO_RECORDS, 'atomic' => false]);
+        $client->addAction('new_client', ['scope' => Model\Action::SCOPE_NONE, 'atomic' => false]);
```

Comment:
NS seems wrong, it should be `Model\UserAction`

Reply:
Fixed now, thanks